src/ufo.rs

```diff
@@ -97,15 +117,55 @@ impl Ufo {
         fn load_impl(path: &Path) -> Result<Ufo, Error> {
             let meta_path = path.join(METAINFO_FILE);
             let meta: MetaInfo = plist::from_file(meta_path)?;
-            let font_path = path.join(FONTINFO_FILE);
-            let font_info = if font_path.exists() {
-                let font_info: FontInfo = plist::from_file(font_path)?;
+            let fontinfo_path = path.join(FONTINFO_FILE);
+            let font_info = if fontinfo_path.exists() {
+                let font_info: FontInfo = plist::from_file(fontinfo_path)?;
                 font_info.validate()?;
                 Some(font_info)
             } else {
                 None
             };
 
+            let lib_path = path.join(LIB_FILE);
+            let lib = if lib_path.exists() {
+                // Value::as_dictionary(_mut) will only borrow the data, but we want to own it.
+                match plist::Value::from_file(lib_path)? {
+                    plist::Value::Dictionary(dict) => Some(dict),
+                    _ => None, // XXX: Return error
```

Comment:
I've opened https://github.com/ebarnard/rust-plist/pull/48 to add `into_dictionary`; maybe reference that issue here, and we can change this if that patch is merged?